safe/common/parameters/test/example_group_select.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+"""Example usage of group select parameter."""
+
+import sys
+from collections import OrderedDict
+from safe.test.utilities import get_qgis_app
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from PyQt4.QtGui import QApplication, QWidget, QGridLayout
+
+from parameters.qt_widgets.parameter_container import ParameterContainer
+
+from safe.common.parameters.group_select_parameter import (
+    GroupSelectParameter)
+from safe.common.parameters.group_select_parameter_widget import (
+    GroupSelectParameterWidget)
+from safe.definitions.constants import (
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->